### PR TITLE
Feature: Opacity Icon

### PR DESCRIPTION
**Pull request by yaira2**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #issue...

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [x] Tested the changes for accessibility

**Screenshots (optional)**
Add screenshots here.

### CODE REVIEW

File: src/Files.App/ResourceDictionaries/PathIcons.xaml [hunk 1]
```diff
@@ -1,4 +1,439 @@
...
+		<ResourceDictionary>
+			<ResourceDictionary.ThemeDictionaries>
+				<ResourceDictionary x:Key="Light">
+					<SolidColorBrush x:Key="IconBaseBrush" Color="#E0000000" />
+					<SolidColorBrush
+						x:Key="IconBaseFadedBrush"
+						Opacity=".12"
+						Color="#E0000000" />
+					<SolidColorBrush
+						x:Key="IconBaseSublteBrush"
```
Comment (chingucoding) [inline src/Files.App/ResourceDictionaries/PathIcons.xaml:15]: Typo

File: src/Files.App/ResourceDictionaries/PathIcons.xaml [hunk 2]
```diff
@@ -1,4 +1,439 @@
...
+						Color="#F0FFFFFF" />
+				</ResourceDictionary>
+				<ResourceDictionary x:Key="Dark">
+					<SolidColorBrush x:Key="IconBaseBrush" Color="#E0FFFFFF" />
+					<SolidColorBrush
+						x:Key="IconBaseFadedBrush"
+						Opacity=".12"
+						Color="#E0FFFFFF" />
+					<SolidColorBrush
+						x:Key="IconBaseSublteBrush"
```
Comment (chingucoding) [inline src/Files.App/ResourceDictionaries/PathIcons.xaml:47]: Typo

File: src/Files.App/ResourceDictionaries/PathIcons.xaml [hunk 3]
```diff
@@ -1,4 +1,439 @@
...
+					<SolidColorBrush x:Key="IconAccentContrastBrush" Color="Black" />
+				</ResourceDictionary>
+				<ResourceDictionary x:Key="HighContrast">
+					<SolidColorBrush x:Key="IconBaseBrush" Color="#E0000000" />
+					<SolidColorBrush
+						x:Key="IconBaseFadedBrush"
+						Opacity=".12"
+						Color="#E0000000" />
+					<SolidColorBrush
+						x:Key="IconBaseSublteBrush"
```
Comment (chingucoding) [inline src/Files.App/ResourceDictionaries/PathIcons.xaml:80]: Typo

File: src/Files.App/ResourceDictionaries/PathIcons.xaml [hunk 4]
```diff
@@ -1,4 +1,439 @@
...
+
+			<Style x:Key="New" TargetType="ContentControl">
```
Comment (chingucoding) [inline src/Files.App/ResourceDictionaries/PathIcons.xaml:111]: I think this key and the keys below could be a bit more descriptive, "New" doesn't tell much about its purpose.

Comment (chingucoding) [reply src/Files.App/ResourceDictionaries/PathIcons.xaml:111]: Also, is the target type correct? I would expect this to be OpacityIcon.

File: src/Files.App/UserControls/OpacityIcon.xaml.cs [hunk 5]
```diff
@@ -0,0 +1,39 @@
+using CommunityToolkit.WinUI.UI;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+
+// The User Control item template is documented at https://go.microsoft.com/fwlink/?LinkId=234236
+
+namespace Files.App.UserControls
+{
+	public sealed partial class OpacityIcon : ContentControl
```
Comment (chingucoding) [inline src/Files.App/UserControls/OpacityIcon.xaml.cs:9]: Do we need to inherit from Content control? I think a "regular" templated control would be better for performance.

**Review by chingucoding — COMMENTED** [review]
Very cool idea, I think it's a good approach to creating those icons.

File: src/Files.App/ResourceDictionaries/PathIcons.xaml (hunk 4 above)
Comment (yaira2) [reply src/Files.App/ResourceDictionaries/PathIcons.xaml:111]: Either one works

File: src/Files.App/UserControls/OpacityIcon.xaml.cs (hunk 5 above)
Comment (yaira2) [reply src/Files.App/UserControls/OpacityIcon.xaml.cs:9]: `Control`?

File: src/Files.App/ResourceDictionaries/PathIcons.xaml (hunk 2 above)
Comment (yaira2) [reply src/Files.App/ResourceDictionaries/PathIcons.xaml:47]: Fixed

File: src/Files.App/ResourceDictionaries/PathIcons.xaml (hunk 1 above)
Comment (yaira2) [reply src/Files.App/ResourceDictionaries/PathIcons.xaml:15]: Fixed

File: src/Files.App/ResourceDictionaries/PathIcons.xaml (hunk 3 above)
Comment (yaira2) [reply src/Files.App/ResourceDictionaries/PathIcons.xaml:80]: Fixed

File: src/Files.App/ResourceDictionaries/PathIcons.xaml (hunk 4 above)
Comment (yaira2) [reply src/Files.App/ResourceDictionaries/PathIcons.xaml:111]: How about starting the key with `ColorIcon...`?

Comment (yaira2) [reply src/Files.App/ResourceDictionaries/PathIcons.xaml:111]: It's going to be renamed once we finish adding all the icons and we remove the existing ColoredIcon control.

File: src/Files.App/UserControls/OpacityIcon.xaml.cs (hunk 5 above)
Comment (chingucoding) [reply src/Files.App/UserControls/OpacityIcon.xaml.cs:9]: Yes

File: src/Files.App/ResourceDictionaries/PathIcons.xaml (hunk 4 above)
Comment (chingucoding) [reply src/Files.App/ResourceDictionaries/PathIcons.xaml:111]: Yeah I think starting with ColorIcon or whatever the final name of the control will be is a good approach.

File: src/Files.App/UserControls/OpacityIcon.xaml.cs (hunk 5 above)
Comment (yaira2) [reply src/Files.App/UserControls/OpacityIcon.xaml.cs]: I switched it to `Control`.

File: src/Files.App/ResourceDictionaries/PathIcons.xaml (hunk 4 above)
Comment (yaira2) [reply src/Files.App/ResourceDictionaries/PathIcons.xaml]: I renamed the icons.

**Review by chingucoding — APPROVED** [review]
LGTM

**Pull request merged**